tests/main/views/test_search.py

```diff
@@ -265,10 +271,10 @@ def test_should_render_summary_with_a_group_of_1_boolean_filter(self):
             '/g-cloud/search?q=email&lot=cloud-software&phoneSupport=true')
         assert res.status_code == 200
         summary = self.find_search_summary(res.get_data(as_text=True))[0]
-        assert '<span class="search-summary-count">1</span> result found' \
-            ' containing <em>email</em> in' \
-            ' <em>Cloud software</em>' \
-            ' where user support is available by <em>phone</em>' in summary
+        assert '<span class="app-search-summary__count">1</span> result found' \
```

Comment:
would triple quotes work for these rather than using `\` to line break?

Reply:
Doing something like:

```
assert """<span class="app-search-summary__count">1</span> result found containing
     <strong>email</strong> in <strong>Cloud software</strong>""" in summary
```

Results in an error:

```
assert '<span class="app-search-summary__count">1</span> result found containing\n             <strong>email</strong> in <strong>Cloud software</strong>' in '<span class="app-search-summary__count">1</span> result found containing <strong>email</strong> in <strong>Cloud software</strong>'
```
Probably my lack of Python knowledge - is there a way to strip the newline from the string?

Reply:
Looks like either I need to `.replace` them, or add a `\` within triple quotes? 

Is there a best practice?

Reply:
Done - wrapped 'em all in parentheses.